src/main/java/com/bittercode/util/DBUtil.java

```diff
@@ -6,10 +6,14 @@
 
 import com.bittercode.constant.ResponseCode;
 import com.bittercode.model.StoreException;
+import org.checkerframework.checker.nullness.qual.Nullable;
 
-public class DBUtil {
 
-    private static Connection connection;
+// Suppressing warnings here because the static block handles all exceptions and
+// potential null parameters/references via the catch clause

```

Comment:
I don't really buy this argument, since the catch clause doesn't catch `NullPointerException`.